packages/grafana-e2e/src/flows/addDataSource.ts

```diff
@@ -32,7 +50,39 @@ export const addDataSource = (config?: Partial<AddDataSourceConfig>): any => {
 
   e2e.pages.DataSource.name().clear();
   e2e.pages.DataSource.name().type(name);
+
+  if (basicAuth) {
+    e2e()
+      .contains('label', 'Basic auth')
```

Comment:
can we use aria-label for this like for other selectors

Reply:
Yes, there are many places that need such updates. I've been postponing it for a while now to do them "all" at once. Probably after the [API redesign](https://github.com/grafana/grafana/issues/25166) to benefit from compound selectors.